### ISSUE TITLE
Feature Request: Small solid circle line end

### ISSUE THREAD

**Issue by mryndzionek** [umlet/umlet#706]
Would it be possible to add small solid circle line end? For 'simulating' connections/junctions.

**Comment by afdia** [comment]
Hi!
You mean a smaller circle than the one in the provde-require type?
![grafik](https://user-images.githubusercontent.com/8770707/221196570-fa74f745-b0cf-48e1-89ce-a970e2e0054d.png)

Is there an UML element/description for this type of relation ending?

**Comment by mryndzionek** [comment]
Yes, smaller and solid. Like in a electronics schematic:
![junct](https://user-images.githubusercontent.com/786191/221203861-c968f7ce-3a60-4584-a38b-691770fbb99b.png)

> Is there an UML element/description for this type of relation ending?

I don't think there is...

**Comment by afdia** [comment]
Correct, there is no such line end at the moment. I mark the issue as possible enhancement for the future
Thanks
